app/javascript/application.js

```diff
@@ -1,3 +1,15 @@
 // Entry point for the build script in your package.json
 import "@hotwired/turbo-rails"
 import "./controllers"
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import store from './redux/store';
+import App from './components/greeting';
+
+ReactDOM.render(
+  <Provider store={store}>
+    <App />
+  </Provider>,
+  document.getElementById('root')
+);
```

Comment:
- [ ] As the previous reviewer indicated your choice of implementing the `React Router` here and advised that you continue with other requirements, I am also unable to access your `Greetings` as the page seems not to be found. Kindly check the project requirements and crosscheck your code to be sure your app opens. Attached is a  screenshot of my when I start the server after setting up the database and seeding some messages in.

![image](https://github.com/gersonhiggins/my_new_rails_app/assets/99715640/8f1d8501-7dee-4a3b-bf06-817ea04ee785)